pppd/sha1.c

```diff
@@ -23,6 +23,8 @@
 #include <string.h>
 #include <time.h>
 #include <netinet/in.h>	/* htonl() */
+#include <sys/types.h>
```

Comment:
Why not put this line in sha1.h

Reply:
There are symbols cannot be found like https://github.com/ppp-project/ppp/blob/c0d579b/pppd/sha1.h#L25

Reply:
I've dropped the change from the patch-set. Seems like the best way is to try to document what compile issue you are seeing and let us figure out the best solution. Please file a bug as @thesamesam suggested.

Generally, I am not a fan of putting includes in header files.